### PR TITLE
fix: update alpine base image

### PR DESCRIPTION
**Pull request by fengshunli**
Signed-off-by: fsl <1171313930@qq.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,12 +17,10 @@ COPY vendor/ vendor/
 RUN CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -a -o manager main.go \
   && CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -a -o daemon ./cmd/daemon/main.go
 
-# Use Ubuntu 20.04 LTS as base image to package the binaries
-FROM ubuntu:focal
-# This is required by daemon connnecting with CRI
-RUN ln -s /usr/bin/* /usr/sbin/ && apt-get update -y \
-  && apt-get install --no-install-recommends -y ca-certificates \
-  && apt-get clean && rm -rf /var/log/*log /var/lib/apt/lists/* /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old
+FROM alpine:3.17
```
Comment (StevenLeiZhang) [inline Dockerfile:20]: I suggest to use 
```
ARG BASE_IMAGE
```
and set its default value is alpine:3.17, then user could customize the base image.

**Comment by zmberg** [comment]
@fengshunli You should solve the failed checks, and confirm all checks are successful.

File: Dockerfile [hunk 2]
```diff
@@ -17,12 +19,10 @@ COPY vendor/ vendor/
 RUN CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -a -o manager main.go \
   && CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -a -o daemon ./cmd/daemon/main.go
 
-# Use Ubuntu 20.04 LTS as base image to package the binaries
-FROM ubuntu:focal
-# This is required by daemon connnecting with CRI
-RUN ln -s /usr/bin/* /usr/sbin/ && apt-get update -y \
-  && apt-get install --no-install-recommends -y ca-certificates \
-  && apt-get clean && rm -rf /var/log/*log /var/lib/apt/lists/* /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old
+FROM ${BASE_IMAGE}:${BASE_IMAGE_VERSION}
```
Comment (zmberg) [inline Dockerfile:22]: Format failed, I suggest you rewrite again.

File: Dockerfile [hunk 3]
```diff
@@ -17,12 +19,10 @@ COPY vendor/ vendor/
...
+
+RUN apk --no-cache add  ca-certificates=20230106-r0 \
```
Comment (zmberg) [inline Dockerfile:24]: RUN apk add --no-cache ca-certificates bash expat ?

File: Dockerfile_multiarch [hunk 4]
```diff
@@ -14,16 +16,18 @@ COPY pkg/ pkg/
 COPY vendor/ vendor/
 
 # Build
-ARG TARGETOS TARGETARCH
+ARG TARGETOS
+ARG TARGETARCH
```
Comment (zmberg) [inline Dockerfile_multiarch:20]: Remove it ?

**Pull request merged**